pymc/tests/test_transforms.py

```diff
@@ -557,3 +557,19 @@ def test_interval_transform_raises():
         tr.Interval(at.constant(5) + 1, None)
 
     assert tr.Interval(at.constant(5), None)
+
+
+def test_transforms_ordered():
+    COORDS = {"question": np.arange(10), "thresholds": np.arange(4)}
+
+    with pm.Model(coords=COORDS) as model:
+        kappa = pm.Normal(
+            "kappa",
+            mu=[-3, -1, 1, 2],
+            sigma=1,
+            dims=["question", "thresholds"],
+            transform=pm.distributions.transforms.ordered,
+        )
```

Comment:
We don't need to use coords for this example.

```suggestion
def test_transforms_ordered():

    with pm.Model() as model:
        kappa = pm.Normal(
            "kappa",
            mu=[-3, -1, 1, 2],
            sigma=1,
            size=(10, 4),
            transform=pm.distributions.transforms.ordered,
        )
```

Also should test with the other transform that you have changed.

Reply:
Done : )
but what about the other tests which are falling ?